crates/ruff/src/rules/flake8_django/rules/locals_in_render_function.rs

```diff
@@ -43,11 +45,25 @@ impl Violation for DjangoLocalsInRenderFunction {
 }
 
 /// DJ003
+impl Analyzer<ast::ExprCall> for DjangoLocalsInRenderFunction {
+    fn rule() -> Rule {
+        Rule::DjangoLocalsInRenderFunction
+    }
+
+    fn run(diagnostics: &mut Vec<Diagnostic>, context: &RuleContext, node: &ast::ExprCall) {
+        locals_in_render_function(diagnostics, context, node);
```

Comment:
(I would suggest inlining this logic and removing functions like `locals_in_render_function` entirely.)